src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+        int randomInt = new Random().nextInt(100);
+        return new Ball(new ColorSupplier().getRandomColor(),randomInt);
+    }

```

Comment:
Please extract Random and ColorSupplier to class variables.
With current implementation we will create new objects for each method call but this is not required. Single object may provide us lots of numbers or colors